stac_geoparquet/pgstac_reader.py

```diff
@@ -7,22 +7,25 @@
 import itertools
 import logging
 import textwrap
-from typing import Any
+from typing import Any, Literal
 
 import dateutil.tz
 import fsspec
+import orjson
 import pandas as pd
-import pyarrow.fs
 import pypgstac.db
 import pypgstac.hydration
 import pystac
 import shapely.wkb
 import tqdm.auto
+from tenacity import before_sleep_log, retry, stop_after_attempt, wait_fixed
```

Comment:
Is this a new dependency? I don't see it in https://github.com/stac-utils/stac-geoparquet/blob/57229c036d9784bfe9c75a7480dea06b77db8826/pyproject.toml#L13-L27